repos/spack_repo/builtin/packages/libvdwxc/package.py

```diff
@@ -31,6 +32,10 @@ class Libvdwxc(AutotoolsPackage):
     depends_on("mpi@2:", when="+mpi")
     depends_on("pfft", when="+pfft")
 
+    depends_on("autoconf", type="build")
+    depends_on("automake", type="build")
+    depends_on("libtool", type="build")
```

Comment:
Are these needed only for/from version `0.5.0`? The `configure` script is not packaged? If yes, we should add the `when=` clause.

Maybe you had to add them when building from `master`? If this is the case, then it should probably be as follows?

```suggestion
    depends_on("autoconf", type="build", when="@master")
    depends_on("automake", type="build", when="@master")
    depends_on("libtool", type="build", when="@master")
```
(See https://spack.readthedocs.io/en/latest/build_systems/autotoolspackage.html#build-system-dependencies)